security/tor-devel/Portfile

```diff
@@ -4,8 +4,7 @@ PortSystem          1.0
 
 name                tor-devel
 conflicts           tor
-version             0.4.1.5
-revision            1
+version             0.4.2.1-alpha
```

Comment:
These days we tend to leave `revision 0` in our files (yes, this is a change from how things were done up until recently.)

Reply:
Fixed.